packages/browser/src/redux/BacktraceReduxMiddleware.ts

```diff
@@ -0,0 +1,32 @@
+import type { Middleware, Action } from 'redux';
+import { BacktraceClient } from '../BacktraceClient';
+
+/**
+ *
+ * @param client BacktraceClient used to send breadcrumbs
+ * @param interceptAction A function that can be used to skip an action or filter out information from a dispatched action, such as PII, that shouldn't be sent to Backtrace. Return undefined to skip an action, otherwise whatever (potentially modified) action returned will be sent to Backtrace.
+ */
+export const createBacktraceReduxMiddleware = (
+    client: BacktraceClient,
+    interceptAction: (action: Action) => Action | undefined = (action) => action,
+) => {
+    if (!client) {
+        throw new Error('Must pass a BacktraceClient to the BacktraceReduxMiddleware.');
+    }
+    const middleware: Middleware = (store) => (next) => (action: Action) => {
```

Comment:
The store variable is unused. If for any reason we have to keep it, can we add a linter ignore?

Reply:
Fixed